packages/ui/src/components/va-modal/VaModal.vue

```diff
@@ -140,6 +149,8 @@ export default defineComponent({
     withoutTransitions: { type: Boolean as PropType<boolean>, default: false },
     overlay: { type: Boolean as PropType<boolean>, default: true },
     overlayOpacity: { type: [Number, String] as PropType<number | string>, default: 0.6 },
+    blur: { type: Boolean as PropType<boolean>, default: false },
```

Comment:
I think having one prop for blur would be enough.
```
<va-modal blur />

<va-modal blur="1.5rem" />
```

Reply:
Let's have a guide in docs, where users can find how to change `-va-blur-radius` CSS variable, instead of making this prop as string.